### PR TITLE
Log service

### PR DESCRIPTION
**Pull request by maeb**

### CODE REVIEW

File: bases/veidemann-log-service-schema-job/job.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
```
Comment (johnerikhalse) [inline bases/veidemann-log-service-schema-job/job.yaml:1]: Any reason the schema initialization is not in same directory as log-service?

File: bases/veidemann-metrics/deployment.yaml [hunk 2]
```diff
@@ -20,47 +20,14 @@ spec:
...
-            - name: DB_HOST
-              value: "rethinkdb-cluster"
-            - name: DB_PORT
-              value: "28015"
-            - name: DB_NAME
-              value: "veidemann"
-            - name: FRONTIER_HOST
-              value: "veidemann-frontier"
-            - name: FRONTIER_PORT
-              value: "7700"
```
Comment (johnerikhalse) [inline bases/veidemann-metrics/deployment.yaml:48]: Where do the metrics server gets infomation from if it is not consulting db or Frontier anymore?

**Review by johnerikhalse — COMMENTED** [review]
Looks good, but have two questions to the code.

File: bases/veidemann-metrics/deployment.yaml (hunk 2 above)
Comment (maeb) [reply bases/veidemann-metrics/deployment.yaml:48]: Agree that it should be explicit. Will fix.

**Pull request merged**